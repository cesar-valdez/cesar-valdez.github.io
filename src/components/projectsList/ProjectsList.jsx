import React from 'react';

import './projectsList.scss';
import Project from './project/Project';

const ProjectsList = () => {
    return (
			<div className="pi_Card-projectsList">
				<span className="pi_Card-projectsList-title">Personal projects</span>
				<ul>
					<Project link={"https://inviesible.netlify.com/"} name={"Inviesible"} description={"Guitar landingpage (HTML, SASS, JADE, JS)"} />
					<Project link={"https://kit-buttons.netlify.com/"} name={"kit buttons"} description={"Set of buttons of different color, type and size (HTML, SASS)"} />
				</ul>
			</div>
    )
}

export default ProjectsList;