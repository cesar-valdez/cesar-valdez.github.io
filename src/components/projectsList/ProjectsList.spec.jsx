import React from 'react';
import { shallow } from 'enzyme';

import ProjectsList from './ProjectsList';

describe('ProjectsList', () => {
    const projectsList = shallow(<ProjectsList/>);

    it('should render projectList component', () => {
        expect(projectsList).toMatchSnapshot();
    });
});