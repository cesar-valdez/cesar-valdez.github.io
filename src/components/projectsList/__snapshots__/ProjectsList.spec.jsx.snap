// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectsList should render projectList component 1`] = `
<div
  className="pi_Card-projectsList"
>
  <span
    className="pi_Card-projectsList-title"
  >
    Personal projects
  </span>
  <ul>
    <Project
      description="Guitar landingpage (HTML, SASS, JADE, JS)"
      link="https://inviesible.netlify.com/"
      name="Inviesible"
    />
    <Project
      description="Set of buttons of different color, type and size (HTML, SASS)"
      link="https://kit-buttons.netlify.com/"
      name="kit buttons"
    />
  </ul>
</div>
`;
